pycqed/measurement/pulse_sequences/fluxing_sequences.py

```diff
@@ -299,19 +306,27 @@ def chevron_seqs(qbc_name, qbt_name, qbr_name, hard_sweep_dict, soft_sweep_dict,
     if 'pulse_length' in hard_sweep_dict:
         max_flux_length = max(hard_sweep_dict['pulse_length']['values'])
         ro_pulses[0]['ref_pulse'] = 'chevron_pi_qbc'
-        ro_pulses[0]['pulse_delay'] = \
-            max_flux_length + flux_pulse.get('buffer_length_start', 0) + \
-            flux_pulse.get('buffer_length_end', 0)
+        ro_pulses[0]['pulse_delay'] = num_cz_gates * \
+            (max_flux_length + flux_pulse.get('buffer_length_start', 0) + \
+            flux_pulse.get('buffer_length_end', 0) + \
+            2*flux_pulse.get('flux_buffer_length', 0) + \
+            2*flux_pulse.get('flux_buffer_length2', 0))
```

Comment:
- add comment that this applies to FLIP only
- is the factor "2*" always correct (NZ vs unipolar?)

Reply:
Indeed, this has to be identical with what is done for the cphase sequence! Is now fixed!